deploy/prod.yaml

```diff
@@ -34,3 +34,35 @@ spec:
   prometheus:
     enabled: true
     path: /metrics
+  vault:
+    enabled: true
+    paths:
+      - mountPath: /var/run/secrets/nais.io/vault
+        kvPath: /kv/prod/fss/medlemskap-oppslag/default
+  secureLogs:
+    enabled: true
+  webproxy: true
+
+  env:
+    - name: AZURE_TENANT
+      value: 62366534-1ec3-4962-8869-9b5535279d0b
```

Comment:
Dette er ikke en hemmelighet?

Reply:
Nei. Den ligger åpent på dokumentasjon av azure. Felles for alle prod-apper.